.github/workflows/ansible-lint-sap_ha_pacemaker_cluster.yml

```diff
@@ -24,15 +24,19 @@ jobs:
 
     steps:
       - name: Check out the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@main
 
       - name: Set up Python 3
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@main
```

Comment:
```suggestion
        uses: actions/setup-python@v4
```

Reply:
Closing for now, see https://github.com/sap-linuxlab/community.sap_install/pull/419#discussion_r1270607762.